db/migrate/20220719155700_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
```

Comment:
- [ ] I am facing the following error while running migration. Kindly fix it.
![Screenshot from 2022-07-20 02-50-01](https://user-images.githubusercontent.com/65860678/179872422-38aa8bb1-0ec0-4200-9f4a-9e7805eaa7d0.png)

Reply:
Hello @uwadonat, the add_user_ref_to_posts.rb migration is not part of the project requirements. I have implemented the requirements that were given by Microverse. The migration you have attempted to run was run in the project that was 2 weeks ago. That was already approved and the relation exists.

The highlighted migration has already worked on my PC and I am able to log in and out as well as create a user account. I have attached screenshot
![2](https://user-images.githubusercontent.com/97033479/179914302-fcf72520-844d-47dc-8d18-0264cbd2d2e8.PNG)
![3](https://user-images.githubusercontent.com/97033479/179914312-359fd402-193e-4919-9a55-b858f7cef83a.PNG)
![4](https://user-images.githubusercontent.com/97033479/179914313-796b534b-05d7-4925-822c-cf21e21a4f4b.PNG)
[IpserLabCofounderAgreementTafaraMafemba.pdf](https://github.com/tafaramafemba/blog-rails/files/9147142/IpserLabCofounderAgreementTafaraMafemba.pdf)
![1](https://user-images.githubusercontent.com/97033479/179914328-a9cbe918-c256-4337-9283-3d324156da6a.PNG)
s

Reply:
Nice! Thanks.